online_store/gt_sports.html

```diff
@@ -0,0 +1,100 @@
+ <!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="gt_sports.css">
+    <title>goodTeens sports online store</title>
+  </head>
+  <body>
+<h2 class="column" id="h2title">goodTeens - Sports</h2>
+<h2 class="column" id="h2title2">Online Store</h2>
+<br><br><br><br><br>
+<p></p>
+<section class="card" id="product1"> 
+  <h3 class="productName1"></h3>
+</section>
+<section class="card" id="product2"> 
+  <h3 class="productName2"></h3>
+</section>
+<section class="card" id="product3"> 
+  <h3 class="productName3"></h3>
+</section>
+<section class="card" id="product4"> 
+  <h3 class="productName4"></h3>
+</section>
+
+<script>
+const sportProducts = [
+  {
+    imgSrc: "/skateboard.jpg",
```

Comment:
Since your images are in the same folder you can refer to them just with a name:
`imgSrc: "skateboard.jpg",` and the same for the others

Reply:
Yep, worked.